ts-obs/cmd/chunkIntervalSet.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+    "context"
+    "errors"
+	"fmt"
+    "os"
+    "strconv"
+    "time"
+
+	"github.com/spf13/cobra"
+    "github.com/jackc/pgx/v4/pgxpool"
+)
+
+// chunkIntervalSetCmd represents the chunk-interval set command
+var chunkIntervalSetCmd = &cobra.Command{
+	Use:   "set <metric> <duration>",
+	Short: "Sets chunk interval in minutes for a specific metric",
+	RunE:  chunkIntervalSet,
+}
+
+func init() {
+	chunkIntervalCmd.AddCommand(chunkIntervalSetCmd)
+}
+
+func chunkIntervalSet(cmd *cobra.Command, args []string) error {
+    var err error
+
+    if len(args) != 2 {
+        return errors.New("\"ts-obs metrics set-metric-chunk\" requires 2 arguments")
+    }
+
+    if os.Getenv("PGPASSWORD_POSTGRES")  == "" {
+        return errors.New("Password for postgres user must be set in environment variable PGPASSWORD_POSTGRES")
+    }
+
+    metric := args[0]
+    var chunk_interval time.Duration
+    chunk_interval, err = time.ParseDuration(args[1])
+    if err != nil {
+        return err
+    }
+
+    if chunk_interval.Microseconds() < 1 {
+        return errors.New("Chunk interval must be at least 1 microsecond")
+    }
```

Comment:
I know it's possible, but maybe we should limit people from actually setting chunks that last less than a minute?

Reply:
Sounds reasonable, I can't think of any use case where anyone would need to go below a minute.